.github/workflows/renew_tls_certificates.yml

```diff
@@ -0,0 +1,33 @@
+name: Renew TLS Certificates
+
+on:
+  schedule:
+# Every Sunday at 08:00
+    - cron: '0 8 * * 0'
```

Comment:
If this takes the site down, there will be a midnight alert. Better to do this during waking hours.